### ISSUE TITLE
Bug: Population Count (not density)

### ISSUE THREAD

**Issue by Charlotte-Morgan** [inasafe/inasafe#1233]
# Problem

Throughout InaSAFE (maps & reports & real time) the results are presented as "population density".
This is not correct unless the number of affected people or number of people needing evacuation has been divided by area.
# Bug fix solution

replace all references to "population density" with "population count" or "number of people" 
# long term solution (not part of this bug fix)

Ideally InaSAFE should be able to present results as population density but only after IF run in projected coordinates. If population density is eventually used it should be expressed as density per unit of area ie 1000 people per sq km.
# cc

@maning 
@akbargumbira
